build.sh

```diff
@@ -73,7 +73,7 @@ fi
 mkdir -p $WORKSPACE_DIR/build
 cd $WORKSPACE_DIR/build
 
-cmake .. -DCMAKE_BUILD_TYPE=$BUILD_TYPE && make -j$THREADS
+cmake .. -B./ -H../ -DCMAKE_BUILD_TYPE=$BUILD_TYPE && make -j$THREADS
```

Comment:
Thanks for pointing out these undocumented options, though I believe the only time they will affect the build, given that it's already running from the build directory, is if there is an existing in-source build.